pkgs/development/compilers/ispc/default.nix

```
@@ -1,41 +1,29 @@
 { lib, stdenv, fetchFromGitHub, fetchpatch
-, cmake, which, m4, python3, bison, flex, llvmPackages
+, cmake, which, m4, python3, bison, flex, llvmPackages, ncurses
 
   # the default test target is sse4, but that is not supported by all Hydra agents
-, testedTargets ? [ "sse2" ]
+, testedTargets ? [ "sse2-i32x4" ]
 }:
 
 stdenv.mkDerivation rec {
   pname   = "ispc";
-  version = "unstable-2021-04-02";
+  version = "1.18.0";
 
   src = fetchFromGitHub {
     owner  = pname;
     repo   = pname;
-    # ISPC release 1.15.0 doesn't build against LLVM 11.1, only against 11.0. So we
-    # use latest ISPC main branch for now, until they support an LLVM version we have.
-    # https://github.com/ispc/ispc/issues/2027#issuecomment-784470530
-    rev    = "3e8313568265d2adfbf95bd6b6e1a4c70ef59bed";
-    sha256 = "sha256-gvr+VpoacmwQlP5gT4MnfmKdACZWJduVMIpR0YRzseg=";
+    rev    = "v${version}";
+    sha256 = "sha256-eI/zUhZDTd2SiFltjrs8kxvQQOPHpqhArGyOW+och3E=";
   };
 
-  patches = [
-    # Fix cmake error: `Failed to find clang++`
-    # https://github.com/ispc/ispc/pull/2055
-    (fetchpatch {
-      url = "https://github.com/erictapen/ispc/commit/338119b2f4e11fcf0b0852de296c320928e572a2.patch";
-      sha256 = "sha256-+RqDq1LMWomu/K4SgK0Nip47b1RwyM6W0cTSNGD4+m4=";
-    })
-  ];
-
   nativeBuildInputs = [ cmake which m4 bison flex python3 llvmPackages.llvm.dev ];
   buildInputs = with llvmPackages; [
-    llvm llvmPackages.libclang
+    llvm libclang openmp ncurses
```

Technically the same, but a bit clearer.

```suggestion
    libllvm libclang openmp ncurses
```

---

Thanks; fixed.